src/components/SearchBox/SearchBox.jsx

```diff
@@ -195,28 +206,18 @@ class SearchBox extends React.Component {
     const isGAResponseReceived = this.state.isGAResponseReceived;
 
     if (this.isSearchInputValid(searchInputValue)) {
-      // Explicit checks for mobile search
-      if (this.props.type === 'mobile') {
-        if (searchType === 'catalog') {
-          gaSearchLabel = 'Submit Catalog Search';
-          GASearchedRepo = 'Encore';
-          requestUrl = this.setEncoreUrl(searchInputValue, encoreBaseUrl, 'eng');
-        } else if (searchType === 'website') {
-          gaSearchLabel = 'Submit Search';
-          GASearchedRepo = 'DrupalSearch';
-          requestUrl = this.setCatalogUrl(searchInputValue, catalogBaseUrl);
-        }
-      } else {
-        // Explicit checks for desktop search
-        if (searchOptionValue === 'catalog') {
-          gaSearchLabel = 'Submit Catalog Search';
-          GASearchedRepo = 'Encore';
-          requestUrl = this.setEncoreUrl(searchInputValue, encoreBaseUrl, 'eng');
-        } else if (searchOptionValue === 'website') {
-          gaSearchLabel = 'Submit Search';
-          GASearchedRepo = 'DrupalSearch';
-          requestUrl = this.setCatalogUrl(searchInputValue, catalogBaseUrl);
-        }
+      if (searchOptionValue === 'circulatingCatalog') {
+        gaSearchLabel = 'Submit Circulating Catalog Search';
```

Comment:
Forgive me, I'm catching up:  before we had "Encore" and "DrupalSearch" as the repos and now we have "Encore", "Research Catalog" and "DrupalSearch"?  So "Research Catalog" is Vega?  I thought Vega was the combined catalog?  

Also, the work to validate that this new label is showing up in GA-- is that captured in a BRD or acceptance criteria somewhere?

Reply:
NP. You're right. This should change from "Encore" to just "Catalog" to be more generic. Vega will be the new "Catalog" and the "Research Catalog" is NYPL's Research Catalog.
I thought Vega was just the circulating catalog.

I haven't heard back yet regarding if we are still somehow keeping track of GA events, if they are useful, and who is doing that.
For the DS Header, we tracked the same GA events and they can all be found here: https://docs.google.com/spreadsheets/d/1uYg1aa2hGiPMLPRQY8uYPSz2YhmO_FjipSzFNt6KrYw/edit#gid=0

I'll have to update it to include this update.

As for testing, there is a section in the "DS Header QA Plan" for checking GA events that can be used to QA this as well (once it's published and deployed to a QA instance): https://docs.google.com/document/d/1i2jML2hwH8o3amx7IVMKNCINUSU5Pd5WyLbYlXdeI4Q/edit#heading=h.f96nonxsuw90

Reply:
To answer the question, no, that's not in the BRD (still being written) or AC. It made sense to include this update here since both searches were being tracked before.